WormholyTests/WormholyTests.swift

```diff
@@ -0,0 +1,169 @@
+
+        XCTAssertTrue(generated == expected)
+    }
+
+    func test_curl_put() {
+        let generated = RequestModelBeautifier.curlExport(
+            request: RequestModel.fixture(
+                endpoint: .patientDetails,
+                method: .put,
+                token: WormholyTests.authenticatedToken,
+                httpBody: try? JSONSerialization.data(withJSONObject: WormholyTests.payload, options: [])
+            )
+        )
+
+        let expected = "curl -H 'accept: application/json' -H 'accept-language: en-US' -H 'authorization: Bearer \(WormholyTests.authenticatedToken)' -H 'content-type: application/json' -H 'host: app-uk.dev.babylontech.co.uk' -H 'x-app-name: babylon' -H 'x-app-version: 4.7.0' -H 'x-platform: ios' -H 'x-platform-version: 12.4' --data-binary '{\"name\":\"Erich Zann\"}' -X PUT --compressed 'https://app-uk.dev.babylontech.co.uk/api/v1/patients/9541561'"
+
+        XCTAssertTrue(generated == expected)
+    }
+
+    func test_curl_patch() {
+        let generated = RequestModelBeautifier.curlExport(
+            request: RequestModel.fixture(
+                endpoint: .patientDetails,
+                method: .patch,
+                token: WormholyTests.authenticatedToken,
+                httpBody: try? JSONSerialization.data(withJSONObject: WormholyTests.payload, options: [])
+            )
+        )
+
+        let expected = "curl -H 'accept: application/json' -H 'accept-language: en-US' -H 'authorization: Bearer \(WormholyTests.authenticatedToken)' -H 'content-type: application/json' -H 'host: app-uk.dev.babylontech.co.uk' -H 'x-app-name: babylon' -H 'x-app-version: 4.7.0' -H 'x-platform: ios' -H 'x-platform-version: 12.4' --data-binary '{\"name\":\"Erich Zann\"}' -X PATCH --compressed 'https://app-uk.dev.babylontech.co.uk/api/v1/patients/9541561'"
+
+        XCTAssertTrue(generated == expected)
+    }
+
+    func test_curl_delete() {
+        let generated = RequestModelBeautifier.curlExport(
+            request: RequestModel.fixture(endpoint: .regions, method: .delete, token: WormholyTests.authenticatedToken)
+        )
+
+        let expected = "curl -H 'accept: application/json' -H 'accept-language: en-US' -H 'authorization: Bearer \(WormholyTests.authenticatedToken)' -H 'content-type: application/json' -H 'host: app-uk.preprod.babylontech.co.uk' -H 'x-app-name: babylon' -H 'x-app-version: 4.7.0' -H 'x-platform: ios' -H 'x-platform-version: 12.4' -X DELETE --compressed 'https://app-uk.preprod.babylontech.co.uk/api/v1/regions'"
+
+        XCTAssertTrue(generated == expected)
+    }
+}
+
+extension WormholyTests {
+    fileprivate enum HTTPMethod: String {
+        case get
+        case post
+        case put
+        case patch
+        case delete
+
+        var value: String {
+            return self.rawValue.uppercased()
+        }
+    }
+
+    fileprivate enum Endpoints: String {
```

Comment:
I tried to add some of our actual endpoints in order to facilitate comparisons with Charles' output.